foundry.toml

```
@@ -14,6 +14,7 @@ inkSepolia = "https://rpc-gel-sepolia.inkonchain.com"
 basecampTestnet = "https://rpc.basecamp.t.raas.gelato.cloud"
 storyTestnet = "https://aeneid.storyrpc.io"
 arbitrumSepolia = "https://arbitrum-sepolia.drpc.org"
+flow_testnet = "https://testnet.evm.nodes.onflow.org"
```

nit: please change this for consistency
```suggestion
flowTestnet = "https://testnet.evm.nodes.onflow.org"
```